File: BAEKJOON/2869.cpp
#include <stdio.h>

int main()
{
    int A, B, V;
    scanf("%d %d %d", &A, &B, &V);
    V -= A;
    int result = V / (A - B);

    if (V == 0) 
        printf("1");
    else if (V % (A - B) > 0)
        printf("%d", result + 2);
    else
        printf("%d", result + 1);
    return 0;
}

File: BAEKJOON/2869.py
import sys

B, A, V = map(int, sys.stdin.readline().split())
V -= B
result = V // (B - A)
if V == 0:
    print(1)
elif V % (B - A) > 0:
    print(result + 2)
else:
    print(result + 1)